### ISSUE TITLE
JanusGraph Re-Index improvements

### ISSUE THREAD

**Issue by gguttikonda** [JanusGraph/janusgraph#1895]
* Remove the arbitrary retry logic for primary query in StandardScannerExecutor. This sometimes fails if there are too many edges to a vertex.
    * Fix the `doc-updates` metrics count.
    * Print info metrics for every Job iteration

For confirmed bugs, please report:
- Version: 0.4 
- Storage Backend: CQL/ Scylla 
- Mixed Index Backend: ElasticSearch

### Stack Trace (if you have one)

- While the CQLResultSetKeyIterator is still working on getting the next row, below exception gets thrown because of fixed time/retry logic.

```
09:33:24 ERROR org.janusgraph.diskstorage.keycolumnvalue.scan.StandardScannerExecutor  - Exception occurred during job execution: {}
org.janusgraph.diskstorage.TemporaryBackendException: Timed out waiting for next row data - storage error likely
        at org.janusgraph.diskstorage.keycolumnvalue.scan.StandardScannerExecutor.run(StandardScannerExecutor.java:157)
        at java.lang.Thread.run(Thread.java:748)

**Comment by gguttikonda** [comment]
A PR is being worked on for this https://github.com/JanusGraph/janusgraph/pull/1896

**Comment by porunov** [comment]
Fixed in #2066
